examples/reconstruct/iem_example_illconditioned_X.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
do we still need the old notebooks `iem_example_illconditioned_X.ipynb` and `em_example_synthetic_data.ipynb`?